File: modules/core/src/main/java/com/lasagnerd/odin/projectStructure/module/rootTypes/collection/OdinCollectionRootProperties.java
package com.lasagnerd.odin.projectStructure.module.rootTypes.collection;

import lombok.Getter;
import lombok.Setter;
import org.jetbrains.jps.model.ex.JpsElementBase;


@Setter
@Getter
public class OdinCollectionRootProperties extends JpsElementBase<OdinCollectionRootProperties> {
    private String collectionName;

    public OdinCollectionRootProperties(String collectionName) {
        this.collectionName = collectionName;
    }


    public OdinCollectionRootProperties() {
    }
}

File: modules/core/src/main/java/com/lasagnerd/odin/projectStructure/module/rootTypes/collection/OdinCollectionRootProperties.kt
package com.lasagnerd.odin.projectStructure.module.rootTypes.collection

import org.jetbrains.jps.model.ex.JpsElementBase

class OdinCollectionRootProperties : JpsElementBase<OdinCollectionRootProperties?> {
    var collectionName: String? = null

    constructor(collectionName: String?) {
        this.collectionName = collectionName
    }


    constructor()
}
